homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -12,13 +12,16 @@
 from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers.start import async_at_started
 
-from .const import DOMAIN
 from .coordinator import SpeedTestDataCoordinator
 
 PLATFORMS = [Platform.SENSOR]
 
+SpeedTestConfigEntry = ConfigEntry["SpeedTestDataCoordinator"]
```

Comment:
```suggestion
SpeedTestConfigEntry = ConfigEntry[SpeedTestDataCoordinator]
```
The coordinator type is already imported here, thus available at runtime.